fabtests/pytest/efa/test_fork.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+
+@pytest.mark.functional
+@pytest.mark.parametrize("environment_variable", ["FI_EFA_FORK_SAFE", "RDMAV_FORK_SAFE"])
+def test_rdm_fork(cmdline_args, completion_type, environment_variable):
```

Comment:
I think the function name and file name should be `test_fork_support`. 

test_fork is too ambiguous.

Reply:
Done